crates/pallet-evm-swap/Cargo.toml

```diff
@@ -0,0 +1,66 @@
+[package]
+name = "pallet-evm-swap"
```

Comment:
```suggestion
name = "pallet-native-to-evm-swap"
```

I think it makes more sense if we'll be splitting the crates.

Reply:
Got it. Should we call precompile logic as `precompile-evm-to-native-swap` ? Or, just `precompile-evm-swap` is enough ?

Reply:
`precompile-evm-to-native-swap` makes sense for symmetry

Reply:
Done